integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -56,18 +58,18 @@ public void authenticationIsEnabled(int responseCode) {
                 .statusCode(responseCode);
     }
 
-    @Given("^get list of Bridge instances with access token doesn't contain Bridge \"([^\"]*)\"$")
-    public void getEmptyBridges(String bridgeName) {
+    @Given("get list of Bridge instances with access token doesn't contain Bridge")
```

Comment:
Maybe
```suggestion
    @Given("get list of Bridge instances with access token doesn't contain generated Bridge")
```
?
To make clear what Bridge we reference to.

Reply:
@sutaakar Fixed. Thank you.